resolwe_bio/processes/workflows/rnaseq_variant_calling.py

```diff
@@ -0,0 +1,400 @@
+            input={"bam": alignment},
+        )
+
+        split_ncigar = Data.create(
+            process=BioProcess.get_latest(slug="gatk-split-ncigar"),
+            input={
+                "bam": mark_duplicates,
+                "ref_seq": inputs.ref_seq,
+            },
+        )
+
+        input_bqsr = {
+            "bam": split_ncigar,
+            "reference": inputs.ref_seq,
+            "known_sites": [inputs.dbsnp],
+            "read_group": inputs.read_group,
+            "advanced": {
+                "use_original_qualities": True,
+            },
+        }
+
+        if inputs.intervals_bed:
+            input_bqsr["intervals"] = inputs.intervals_bed
+
+        if inputs.indels:
+            input_bqsr["known_sites"].append(inputs.indels)
+
+        bqsr = Data.create(
+            process=BioProcess.get_latest(slug="bqsr"),
+            input=input_bqsr,
+        )
+
+        input_hc = {
+            "alignment": bqsr,
+            "genome": inputs.ref_seq,
+            "dbsnp": inputs.dbsnp,
+            "mbq": 10,
+            "advanced": {
+                "soft_clipped": True,
+                "java_gc_threads": inputs.advanced.java_gc_threads,
+                "max_heap_size": inputs.advanced.max_heap_size,
+            },
+        }
+
+        if inputs.intervals:
+            input_hc["intervals"] = inputs.intervals
+        elif inputs.intervals_bed:
+            input_hc["intervals_bed"] = inputs.intervals_bed
+
+        hc = Data.create(
+            process=BioProcess.get_latest(slug="vc-gatk4-hc"),
+            input=input_hc,
+        )
+
+        input_filtration = {
+            "vcf": hc,
+            "ref_seq": inputs.ref_seq,
+            "filter_expressions": inputs.filter_expressions,
+            "filter_name": inputs.filter_name,
+            "advanced": {
+                "cluster": 3,
+                "window": 35,
+                "java_gc_threads": inputs.advanced.java_gc_threads,
+                "max_heap_size": inputs.advanced.max_heap_size,
+            },
+        }
+
+        filtration = Data.create(
+            process=BioProcess.get_latest(slug="gatk-variant-filtration"),
+            input=input_filtration,
+        )
+
+        if inputs.exclude_filtered:
+            input_select = {
+                "vcf": filtration,
+                "select_type": inputs.select_variants.select_type,
+                "exclude_filtered": True,
+                "advanced_options": {
+                    "java_gc_threads": inputs.advanced.java_gc_threads,
+                    "max_heap_size": inputs.advanced.max_heap_size,
+                },
+            }
+
+            Data.create(
+                process=BioProcess.get_latest(slug="gatk-select-variants"),
+                input=input_select,
+            )
+
+        multiqc_tools = [inputs.reads, alignment, mark_duplicates, bqsr]
+
+        if inputs.preprocessing:
+            multiqc_tools.insert(1, preprocessing)
```

Comment:
Nothing wrong, just FYI: MultiQC process/tool settings define the module order in the MultiQC report. There is no need to pass the objects in any order here.